.github/workflows/preview-env-teardown.yml

```diff
@@ -29,21 +29,14 @@ jobs:
         with:
           credentials_json: ${{ steps.secrets.outputs.PREVIEW_ENV_GCLOUD_SA_KEY }}
 
-      - name: Find deployment status comment
-        uses: peter-evans/find-comment@v3
-        id: find-comment
-        with:
-          issue-number: ${{ github.event.number }}
-          body-includes: <!-- preview-env -->
-
       - name: Update comment with tear-down warning
-        if: steps.find-comment.outputs.comment-id != ''
-        uses: peter-evans/create-or-update-comment@v4
+        uses: mshick/add-pr-comment@v2
         with:
-          comment-id: ${{ steps.find-comment.outputs.comment-id }}
-          body: |
-            :warning: Preview environment for commit ${{ github.sha }} is being deleted. This usually takes 2-3 minutes. <!-- preview-env -->
-          edit-mode: replace
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message-id: preview-env-status
```

Comment:
`message-id` is also not necessary, if you don't plan on having more than one comment you want to update throughout the process. There's a default value like `add-pr-comment`.